DOC Prepare for docs site (#220)
silverstripe/silverstripe-login-forms: docs/en/index.md

---
title: Login forms
summary: Simplifying the login forms for a SilverStripe site

> icon: file-lines

Reviewer: This icon doesn't exist in font awesome v5: https://fontawesome.com/v5/search?q=file-lines&o=r
You may have meant `file-alt`: https://fontawesome.com/v5/search?q=file-alt&o=r

Author: Updated

---

# Login forms

> Thos module provides an improved default view for the authentication UIs in SilverStripe.

Reviewer: ```suggestion
This module provides an improved default view for the authentication UIs in Silverstripe CMS.
```

Author: Updated

Rather than applying the forms to the site's default Page template, Login Forms renders a simple, responsive, standalone interface that aligns with the design of the CMS.

> This can be tweaked to suit an individual SilverStripe site, but doesn't require any extra work out of the box to incorporate the UI into the site's design.

Reviewer: ```suggestion
This can be tweaked to suit an individual Silverstripe CMS site, but doesn't require any extra work out of the box to incorporate the UI into the site's design.
```

Author: Updated

---
title: Login forms
summary: Simplifying the login forms for a Silverstripe site
icon: file-alt
---

# Login forms

> Thos module provides an improved default view for the authentication UIs in Silverstripe CMS.

Reviewer: ```suggestion
This module provides an improved default view for the authentication UIs in Silverstripe CMS.
```
as per https://github.com/silverstripe/silverstripe-login-forms/pull/220#discussion_r2220713394

Author: Updated
